<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>891</width>
      <height>491</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Line Graph</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Scatter Graph</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>901</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen_Megasquirt_Log"/>
    <addaction name="actionOpen_LibreEMS_Log"/>
    <addaction name="actionOpen_Embedded_Lockers_Log"/>
    <addaction name="actionClose_Log"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuExport">
    <property name="title">
     <string>Export</string>
    </property>
    <addaction name="actionExport_as_CSV"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuExport"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen_Log">
   <property name="text">
    <string>&amp;Open Log</string>
   </property>
  </action>
  <action name="actionClose_Log">
   <property name="text">
    <string>&amp;Close Log</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionSelect_Dialog">
   <property name="text">
    <string>Select Dialog</string>
   </property>
  </action>
  <action name="actionOpen_Megasquirt_Log">
   <property name="text">
    <string>Open Megasquirt Log</string>
   </property>
  </action>
  <action name="actionOpen_LibreEMS_Log">
   <property name="text">
    <string>Open LibreEMS Log</string>
   </property>
  </action>
  <action name="actionOpen_Embedded_Lockers_Log">
   <property name="text">
    <string>Open Embedded-Lockers Log</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExport_as_CSV">
   <property name="text">
    <string>Export as CSV</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
